src/shim.js

```diff
@@ -132,7 +161,13 @@ function executeCommand(command) {
         const target = objectSpace[command.target];
         const method = target[command.name];
 
-        if (haveCallbacks.indexOf(command.name) === -1) {
+        if (command.name === 'addEvent') {
+            addEvent(command.target, command.params[0].type, command.params[0].event, command.params[0].args);
+            completeCommand(command);
+        } else if (command.name === 'removeEvent') {
```

Comment:
OK. I won't have a chance to fix it tonight. I am just leaving work and then traveling tomorrow. If you can get it then I could merge and release a new build. Thanks for awesome job.